virttest/ceph.py

```diff
@@ -167,7 +166,7 @@ def rbd_image_map(ceph_monitor, rbd_pool_name, rbd_image_name):
     """
     cmd = "rbd map %s --pool %s -m %s" % (rbd_image_name, rbd_pool_name,
                                           ceph_monitor)
-    output = decode_to_text(process.system_output(cmd, verbose=True))
+    output = process.system_output(cmd, verbose=True).stdout_text
```

Comment:
Hi, @clebergnu 
I saw the return of `process.system_output` is `cmd_result.stdout,` so I wonder whether `CmdResult.stdout` has the attribute `stdout_text`, from codes, seems don't have it.
my testing:
```
Python 3.6.8 (default, Dec  5 2019, 15:45:45) 
[GCC 8.3.1 20191121 (Red Hat 8.3.1-5)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from avocado.utils import process
>>> process.system_output('uname -r', verbose=True).stdout_text
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'bytes' object has no attribute 'stdout_text'
>>> 
```

Thanks.

Reply:
@YongxueHong this is certainly a mistake, it should be `process.run().stdout_text`.  I'll send a PR fixing that.